service/src/main/java/bio/terra/pipelines/service/PipelineRunsService.java

```diff
@@ -353,4 +353,8 @@ public PageResponse<List<PipelineRun>> findPipelineRunsPaginated(
         CursorBasedPageable.getEncodedCursor(
             pipelineRuns.get(pipelineRuns.size() - 1).getId().toString(), postSlice.hasNext()));
   }
+
+  public long getPipelineRunCount(String userId) {
```

Comment:
can we add a test for this and where its used to return "totalResults" above.  Also is there any reason to put this function in the [pipelineRepository](https://github.com/DataBiosphere/terra-scientific-pipelines-service/blob/main/service/src/main/java/bio/terra/pipelines/db/repositories/PipelineRunsRepository.java) class instead?  the function name woudl be something like `countByUserId`